release.md

```diff
@@ -0,0 +1,26 @@
+# Release instructions
+
+These are the minimum required steps to publish the statful javascript client on npm.  
+Must be executed in the following order
+
+## Locally
+1. update version in package.json
```

Comment:
please update to
1. `npm run test`
2. `npm run build`
3. update version in package.json